apps/dot-voting/app/components/VotingCard.js

```diff
@@ -96,13 +96,14 @@ const VotingCard = ({ vote, onSelectVote }) => {
         {options.length > 2 && (
           <div css="text-align: center; width: 100%; margin-top: 10px">
             <Badge
-              shape="compact"
-              foreground={`${theme.surfaceOpened}`}
-              background={`${theme.surfaceUnder}`}
+              foreground={theme.surfaceContentSecondary}
+              background={theme.surfaceUnder}
+              uppercase={false}
               css={`
                 cursor: pointer;
                 padding: 2px 8px;
                 pointer-events: auto;
+                mix-blend-mode: normal;
```

Comment:
Do we still need this, or is it vestigial?

Reply:
Good catch! Removed